packages/plugin-prisma/tests/index.test.ts

```diff
@@ -65,6 +65,50 @@ describe('prisma', () => {
     `);
   });
 
+  it('skips fields based on @skip and @include directives', async () => {
+    const query = gql`
+      query {
+        me {
+          id
+          name @skip(if: true)
+          email @include(if: false)
```

Comment:
I think you need to add these to a relation instead to see any difference, unless you query a type that is using select mode.  This change only changes the generated query, and not what revolvers get run.  

```graphql
query {
        me {
          id
          profile @skip(if: true) {
            bio
          } 
          posts(limit: 1) @include(if: false) {
            id
          }

Reply:
Thanks! I just updated the test

Reply:
awesome, thanks!